### PR TITLE
chore(dependabot): update dependabot config to update GitHub Action workflows

### PR DESCRIPTION
**Pull request by boahc077**
Signed-off-by: Ashish Kurmi <akurmi@stepsecurity.io>

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Unit tests (`npm test`) were run locally and passed
- [ ] E2E Tests (`npm run test.karma.prod`) were run locally and passed
- [ ] Prettier (`npm run prettier`) was run locally and passed

None of the above

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):
Security improvement - by enabling dependabot for GitHub actions, the bot will automatically update GitHub Actions workflows when a new action version is released. 

## What is the current behavior?
Currently, one needs to manually upgrade GitHub actions to the latest version.

## What is the new behavior?
Once the PR is merged, dependabot will automatically create pull requests to upgrade GitHub actions as updates are released.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Testing

I authored the change based on dependabot documentation available here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

### CODE REVIEW

**Comment by rwaskiewicz** [comment]
Thanks @boahc077!

We're currently waiting to add this functionality to Stencil Core while we add it to [Stencil Store](https://github.com/ionic-team/stencil-store/pull/155/files) and make sure it fits our needs. That way we don't need to go back and update the configuration across multiple Stencil repositories. Once that linked PR has landed, we'll circle back to this.

In the mean time, can you do me a favor and fill out the PR template? Thanks!

**Comment by boahc077** [comment]
Thanks @rwaskiewicz for the prompt review, I just filled out the PR template. Please let me know if there is anything else I can do to help.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -31,3 +31,7 @@ updates:
         versions: ['>7.3.3']
       - dependency-name: '@types/inquirer'
         versions: ['>7.3.1']
+  - package-ecosystem: 'github-actions'
+    directory: /
```
Comment (rwaskiewicz) [inline .github/dependabot.yml:35]: This is a really small nitpick, but I'm going to add quotes around this value here:
```suggestion
    directory: '/'
```

**Comment by rwaskiewicz** [comment]
Merged, thanks @boahc077 !

**Pull request merged**